src/main/java/thaumic/tinkerer/preloader/ThaumicTLoaderContainer.java

```diff
@@ -61,7 +63,7 @@ public boolean registerBus(EventBus bus, LoadController controller) {
 
     @Override
     public String[] getASMTransformerClass() {
-        return new String[]{"thaumic.tinkerer.preloader.AccessTransformer"};
+        return null;
```

Comment:
this looks really dumb why not remove the whole override

Reply:
dunno, seems to work ...

Reply:
Because this is a method from an interface. There is no super to call into.